### PR TITLE
add skeleton to my internetshop

### PR DESCRIPTION
**Pull request by syevtushok**

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/dao/OrderDao.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    Order update(Order item);
+
+    boolean delete(Long id);
+
+    boolean delete(Order item);
+
+    List<Order> getAll();
+
+    List<Order> getAllOrders();
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/dao/OrderDao.java]: what's the difference? I think, you should remove one :)

File: src/main/java/mate/academy/internetshop/dao/UserDao.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package mate.academy.internetshop.dao;
+
+import java.util.List;
+import java.util.Optional;
+
+import mate.academy.internetshop.model.User;
+
+public interface UserDao {
+    User create(User item);
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/dao/UserDao.java]:
```suggestion
    User create(User user);
```

File: src/main/java/mate/academy/internetshop/dao/UserDao.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+    Optional<User> get(Long id);
+
+    User update(User item);
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/dao/UserDao.java]:
```suggestion
    User update(User user);
```

File: src/main/java/mate/academy/internetshop/model/Bucket.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
+package mate.academy.internetshop.model;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Bucket {
+    private User user;
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/model/Bucket.java]:
```suggestion
    private Long userId;
```

File: src/main/java/mate/academy/internetshop/model/Bucket.java [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    private List<Item> items;
+    private Long bucketId;
+    private BigDecimal sumOfMoney;
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/model/Bucket.java]:
```suggestion
    private BigDecimal price;
```

File: src/main/java/mate/academy/internetshop/model/Order.java [hunk 6]
```diff
@@ -0,0 +1,62 @@
+package mate.academy.internetshop.model;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public class Order {
+    private Long orderId;
+    private List<Item> itemList;
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/model/Order.java]:
```suggestion
    private List<Item> items;
```

File: src/main/java/mate/academy/internetshop/service/imp/OrderServiceImpl.java [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+    @Override
+    public List<Order> getUserOrders(User user) {
+        List<Order> orderList = new ArrayList<>();
+        for (Order order : orderDao.getAll()) {
+            if (order.getUserId().equals(user.getId())) {
+                orderList.add(order);
+            }
+        }
+        return orderList;
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/service/imp/OrderServiceImpl.java]: try to use Stream api

File: src/main/java/mate/academy/internetshop/dao/OrderDao.java (hunk 1 above)
Comment (syevtushok) [reply src/main/java/mate/academy/internetshop/dao/OrderDao.java]: done

File: src/main/java/mate/academy/internetshop/dao/UserDao.java (hunk 2 above)
Comment (syevtushok) [reply src/main/java/mate/academy/internetshop/dao/UserDao.java]: copy paste, done))

File: src/main/java/mate/academy/internetshop/dao/UserDao.java (hunk 3 above)
Comment (syevtushok) [reply src/main/java/mate/academy/internetshop/dao/UserDao.java]: copy paste, done))

File: src/main/java/mate/academy/internetshop/model/Bucket.java (hunk 4 above)
Comment (syevtushok) [reply src/main/java/mate/academy/internetshop/model/Bucket.java]: done

File: src/main/java/mate/academy/internetshop/model/Bucket.java (hunk 5 above)
Comment (syevtushok) [reply src/main/java/mate/academy/internetshop/model/Bucket.java]: done

File: src/main/java/mate/academy/internetshop/model/Order.java (hunk 6 above)
Comment (syevtushok) [reply src/main/java/mate/academy/internetshop/model/Order.java]: done

File: src/main/java/mate/academy/internetshop/service/imp/OrderServiceImpl.java (hunk 7 above)
Comment (syevtushok) [reply src/main/java/mate/academy/internetshop/service/imp/OrderServiceImpl.java]: done

**Review by Tarabarnik — APPROVED** [review]
роби squash and merge та приступай до дз 15

**Pull request merged**